src/SFA.DAS.EmployerFinance/Models/LevyDeclarationSaga.cs

```diff
@@ -0,0 +1,95 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using SFA.DAS.EmployerFinance.Messages.Events;
+
+namespace SFA.DAS.EmployerFinance.Models
+{
+    public class LevyDeclarationSaga : Entity
```

Comment:
Yeah I struggled with this one. Initially I went with `LevyDeclarationJob` but we already have the concept of a job in our `SFA.DAS.EmployerFinance.Jobs` project. NServiceBus calls it a Saga but technically it's a Process Manager, out of the two I went with Saga because I figured devs working with NServiceBus would be more familiar with the concept (a long running workflow with state). I guess it back fired. Do you have any idea what we could rename it to?

Reply:
Ah yes, I remember us talking about this now 🙂.

I do think we should avoid Saga or Job, because they're encumbered with specific meaning in this domain. I don't have a definitive suggestion to give, but from reading your comment, you basically called it a Process and a Workflow, so perhaps we could use one of those?

Reply:
I'd prefer to avoid using `WebJob` as it's implementation specific. Lets leave it for now, it's an easy change to rename it later on.